packages/cyberstorm/src/schema/schema.ts

```diff
@@ -36,67 +55,90 @@ export interface Package {
   isPinned?: boolean;
   isNsfw?: boolean;
   isDeprecated?: boolean;
+  categories: Category[]; // Category ids
```

Comment:
Not sure what the comment tries to say since the interface is defined above and contains the name AND a slug?